enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path

mingfeima opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #74023

Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

Add channels last 3d support for both Conv3d and ConvTransposed3d

cc @jgong5 @XiaobingSuper @sanchitintel @ashokei @jingxu10

mingfeima committed:
enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path

[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"




[ghstack-poisoned]

VitalyFedyunin commented:
@VitalyFedyunin has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35782442).

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

[ghstack-poisoned]

VitalyFedyunin commented:
@VitalyFedyunin has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35782442).

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

[ghstack-poisoned]

VitalyFedyunin commented:
@VitalyFedyunin has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35782442).

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

Add channels last 3d support for both Conv3d and ConvTransposed3d

[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

Add channels last 3d support for both Conv3d and ConvTransposed3d

cc jgong5 @XiaobingSuper sanchitintel ashokei jingxu10

[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

Add channels last 3d support for both Conv3d and ConvTransposed3d

cc jgong5 @XiaobingSuper sanchitintel ashokei jingxu10

[ghstack-poisoned]

mingfeima committed:
Update on "enable channels last 3d for Conv3d and ConvTranspose3d on mkldnn path"


Differential Revision: [D35782442](https://our.internmc.facebook.com/intern/diff/D35782442)

Add channels last 3d support for both Conv3d and ConvTransposed3d

cc jgong5 @XiaobingSuper sanchitintel ashokei jingxu10

[ghstack-poisoned]
